web/src/components/navbar/NavbarTab.tsx

```diff
@@ -14,17 +13,10 @@ const NavbarTab: React.FC<NavbarTabProps> = ({
     name,
     path,
 }) => {
-    const router = useRouter();
-
     return (
         <Link
```

Comment:
Nice catch finding a solution for force refresh. I'm fine with either behaviour. Personally I think I would assume click the tab would do nothing (i.e. use cached page) when already on that page, but I think it would be more useful for it to perform a force refresh (I can't think of any specific functionality, though) - it's up to you which you want to implement.

Reply:
(make sure to change the closing tag also)